src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/InformAgreedExtensionDateCallbackHandlerTest.java

```diff
@@ -285,14 +309,16 @@ void shouldReturnExpectedError_whenValuesAreInvalid() {
 
             // Then
             assertThat(response.getErrors())
-                .containsOnly("The agreed extension date must be a date in the future");
+                .contains("The agreed extension date must be a date in the future");
```

Comment:
Please add the other errs in assert as well if we are expecting more then one errors here.